tests/test_end_to_end.py

```diff
@@ -101,4 +101,54 @@ def testVariantsSearch(self):
 
         # TODO: Add test cases for other methods when they are implemented.
 
+    def testCallSetsSearch(self):
```

Comment:
This method is way too long.  If each thing you're testing doesn't depend on the results of the previous test, it would be good to break each of them out into their own method.